common/landed_titles/wc_religious_titles.txt

```diff
@@ -1005,4 +1005,23 @@ d_bronze_wyrmcult = {
 	definite_form = yes
 	ruler_uses_title_name = no
 	landless = yes
-}
\ No newline at end of file
+}
+
+### titanic_group ##
+#d_thunder_king = {
+#	# Juke Note: Originally planned for Shenism to get a head of faith only if the Thunder King returns, and to then remove it again once he dies. Unfortunately, I cant get it to display him as "of the Mogu Empire" - despite adding 'ruler_uses_title_name = no' to the landed title it didnt work. 
```

Comment:
I made that for Kaldorei Empire, you can check if you want.
If not, I can take a look myself.

Reply:
There is a bug with Lei Shen when you are obsing/not controling him.
4 times in raw, when i was obsing the game with the rule "Lei Shen spawn after year 650 guarantee", Lei Shen died "in battle" 1 month after he came.
But it's clearly impossible first because he was not even on an army, then because he has 65 prowess so it's statisticly impossible. You can see my screen shots and few more comments on discord #ck3-testing.
I continue to investigate but i'm not coder.

Reply:
Die in battle is a fallback for if he becomes unlanded. do you have any more info?

Reply:
> I made that for Kaldorei Empire, you can check if you want. If not, I can take a look myself.

Probably not worth it, as it would require me to redo a bunch of code i commented out, The issue was that he still used the head of faith loc even though the mogu use Dynastic Titles (like the arabs)